spec/spec_helper.rb

```diff
@@ -2,6 +2,8 @@
 SimpleCov.start
 require "codeclimate-test-reporter"
 CodeClimate::TestReporter.start
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.